src/foraging_gui/Foraging.py

```diff
@@ -480,7 +480,8 @@ def _GetSettings(self):
             'newscale_port_tower1':'',
             'newscale_port_tower2':'',
             'newscale_port_tower3':'',
-            'newscale_port_tower4':''
+            'newscale_port_tower4':'',
+            'enable_show_log_info_in_console':False,
```

Comment:
Can you make a note in the README about this additional setting?

Reply:
added.